README.md

```diff
@@ -132,10 +146,24 @@ $ DATABASE_NAME=uwazi_e2e INDEX_NAME=uwazi_e2e yarn hot
 On a different console tab, run
 
 ```
-$ yarn e2e-puppeteer
+$ yarn cypress
+```
+
+This will open the Cypress interface where you can select the tests to run. It's recommended that Cypress tests are run with Chrome, or Chrome based browsers.
```

Comment:
```suggestion
This will open the Cypress interface where you can select the tests to run. It's recommended that Cypress tests are run with Chrome or Chrome based browsers.
```